frontend/simple/js/backend/hapi.js

```diff
@@ -73,29 +73,55 @@ export class HapiBackend extends Backend {
   subscriptions () {
     return Object.keys(store.state.contracts)
   }
-  async subscribe (contractId: string) {
+  async subscribeAndSyncIdentity (username: string, ephemeral: bool) {
+    let identity = await namespace.lookup(username)
+    await this.subscribeAndSync(identity, ephemeral)
+  }
+  async unsubscribeIdentity (username: string, ephemeral: bool) {
+    let identity = await namespace.lookup(username)
+    await this.unsubscribe(identity, ephemeral)
+  }
```

Comment:
That makes, sense to me but I wasn't sure what that file should look like at the time. I'll transfer this to ties own beast like a `backend.utils` kind of thing.